#87 Support avif file format for backgrounds

shepard8 opened the pull request:
The parts sfeir-school-theme is responsible for have been fixed.

However, for general slide backgrounds, currently the files need to be named with a file extension allowed by RevealJS.

I opened a ticket on the RevealJS repository (see ticket #87), but this is out of scope of this project, so I guess this branch can be merged already.

jefBinomed reviewed (changes requested):
Il faudrait que tu mettes une image avif pour le prouver :)

shepard8 commented:
Le background que j'ai ajouté est un fichier AVIF mais avec l'extension JPG, j'explique ça dans l'issue 87 😃 

![image](https://user-images.githubusercontent.com/5731430/224087361-1f9f4c8d-9c58-4f25-ac58-eb24d20bee55.png)

shepard8 commented:
Finally, I added the compiled versions to the patch so that files with the avif extension are usable directly.

I renamed the file as it made no sense anymore to use a fake extension.

shepard8 commented:
@jefBinomed Je pense qu'il y a encore un souci avec netlify. Patch-package est appliqué (cfr netlify log) mais le résultat n'est pas pris en compte pour construire le sfere-school-theme.mjs.

![image](https://user-images.githubusercontent.com/5731430/224260200-f2b2a4e6-0342-4f5c-89fc-f5c1b76928dc.png)

![image](https://user-images.githubusercontent.com/5731430/224260276-63fb8938-439d-4df1-b017-b469778d4ef2.png)

Ce qui fait que ni les webp, ni les avif ne sont supportés pour le moment 😞 

Si je crée un compte netlify, je peux y regarder moi-même ? Je ne veux pas te faire perdre de temps ^^

Merci,
